### PR TITLE
Added RHEL8 support

### PR DESCRIPTION
**Pull request by sbluhm**
## What does this PR change?

Added RHEL8 build support.

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: No functionality change

- [X] **DONE**

## Test coverage
- No tests: No code change

- [X] **DONE**

## Links

Fixes #
None

- [X] **DONE**

## Changelogs
- Added RHEL8 support

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: web/spacewalk-web.spec [hunk 1]
```diff
@@ -186,16 +192,15 @@ database.
...
 ln -sf %{nodejs_sitelib} .
 BUILD_VALIDATION=false node build.js
 popd
-%endif
 sed -i -r "s/^(web.buildtimestamp *= *)_OBS_BUILD_TIMESTAMP_$/\1$(date +'%%Y%%m%%d%%H%%M%%S')/" conf/rhn_web.conf
 
 %install
 make -C modules install DESTDIR=$RPM_BUILD_ROOT PERLARGS="INSTALLDIRS=vendor" %{?_smp_mflags}
+sed -i s#/srv/www/htdocs#/%{www_path}/www/htdocs# html/Makefile 
 make -C html install PREFIX=$RPM_BUILD_ROOT
```
Comment (mcalmer) [inline web/spacewalk-web.spec:204]:
```suggestion
make -C html install PREFIX=$RPM_BUILD_ROOT INSTALL_DEST=%{www_path}/www/htdocs
```

**Review by mcalmer — CHANGES_REQUESTED** [review]
I added one suggestion to prevent using sed.

Please also add a line in web/spacewalk-web.changes .
Please do not add a Header. Just one line on top of the file with what you changed.

The Header is added later automatically when we submit for uyuni.

**Comment by mcalmer** [comment]
@sbluhm About changelogs https://github.com/uyuni-project/uyuni/wiki/Contributing#git-changelogs

**Comment by sbluhm** [comment]
Thank you. Added your feedback.

**Review by mcalmer — APPROVED** [review]
Looks good to me. Thanks.

**Pull request merged**